src/back-end/lib/db.ts

```diff
@@ -1591,9 +1592,11 @@ export const awardCWUProposal = tryDb<[Id, string, AuthenticatedSession], CWUPro
     // Update all other proposals on opportunity to Not Awarded
     await connection('cwuProposalStatuses')
       .transacting(trx)
-      .whereIn('proposal', async () => {
-        await connection('cwuProposals')
-          .where({ opportunity: proposalRecord?.opportunity });
+      .whereIn('proposal', async function() {
+        this
+          .select('id')
+          .from('cwuProposals')
+          .where({ opportunity: proposalRecord.opportunity });
```

Comment:
Out of curiosity, why was this change made?

Reply:
The original wasn't working.  On Telegram, Jesse pointed out it was throwing an error when there were other proposals to mark as not awarded.  I needed to use `this` instead of referring to the `connection` instance.

Reply:
ah got it, thanks!